### PR TITLE
fix: Run cSpell for typos

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Review by chrisdavidmills — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
CI failure, I think?

**Comment by mattiapontonio** [comment]
Seems to fail for this

Good
`{{nonExistentMacro(&quot;yada&quot;)}`

Bad
`{{nonExistentMacro(\"yada\")}}`

🤷🏻‍♂️

What I don't understand is where this PR touched that macro.

**Comment by nschonni** [comment]
@chrisdavidmills rebased now that it looks like CI is feeling happier in general again 😄

File: client/src/search.tsx [hunk 1]
```diff
@@ -337,7 +337,8 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
                 {...getItemProps({
                   key: item.url,
                   className:
-                    "result-item " + (i === highlightedIndex ? "highlit" : ""),
+                    "result-item " +
+                    (i === highlightedIndex ? "highlight" : ""),
```
Comment (nschonni) [inline client/src/search.tsx:341]: I renamed the CSS as well, but I wasn't sure if this was intentional for a different reason

**Comment by mattiapontonio** [comment]
Lets talk about this.
If You rebase the CI passes.
Should the CI Say something like "Please rebase"?
Anyway why GitHub don't marks something by default?

**Comment by chrisdavidmills** [comment]
Much better, thanks @nschonni !

**Comment by peterbe** [comment]
I would have liked to review big code changing PRs like this first.

**Pull request merged**